zeq/hbp/vocabulary.h

```diff
@@ -15,6 +16,13 @@ namespace zeq
 namespace hbp
 {
 
+struct imageRaw
+{
+    uint32_t resX;
+    uint32_t resY;
+    uint8_t* data;
```

Comment:
What's the format of data. I can infer from other part of the code that this is a RGBA image, then the struct name should be RGBAImage or ImageRGBA.
State clearly why *data is a plain pointer.